### ISSUE TITLE
Questions about handling of `anyOf`

### ISSUE THREAD

**Issue by finswimmer** [elisiariocouto/jsonschema-markdown#19]
Hey,

thanks a lot for this great project :+1: :pray: 

I have some questions about the current handling of `anyOf`.

This schema:

```json
{
  "properties": {
    "bar": {
      "anyOf": [
        {
          "type": "string"
        },
        {
          "type": "null"
        }
      ],
      "title": "Bar"
    }
  },
  "required": [
    "bar"
  ],
  "title": "Foo",
  "type": "object"
}
```

generates this markdown file:

```md
# Foo

JSON Schema missing a description, provide it using the `description` key in the root of the JSON document.

### Type: `object`

| Property | Type | Required | Possible values | Deprecated | Default | Description | Examples |
| -------- | ---- | -------- | --------------- | ---------- | ------- | ----------- | -------- |
| bar | `string` | ✅ | string |  |  |  |  |


---

Markdown generated with [jsonschema-markdown](https://github.com/elisiariocouto/jsonschema-markdown).
```

As you can see `null` is not mentioned. According to this ToDo in the code it seems to be expected. Can you explain why?

https://github.com/elisiariocouto/jsonschema-markdown/blob/4bc262e277487064f465ab9f8f2bbdaed8fb668b/jsonschema_markdown/converter/markdown.py#L319-320

In the other hand let's replace `"null"` with `"integer"`

```json
{
  "properties": {
    "bar": {
      "anyOf": [
        {
          "type": "string"
        },
        {
          "type": "integer"
        }
      ],
      "title": "Bar"
    }
  },
  "required": [
    "bar"
  ],
  "title": "Foo",
  "type": "object"
}
```

```md
# Foo

JSON Schema missing a description, provide it using the `description` key in the root of the JSON document.

### Type: `object`

| Property | Type | Required | Possible values | Deprecated | Default | Description | Examples |
| -------- | ---- | -------- | --------------- | ---------- | ------- | ----------- | -------- |
| bar | `string` | ✅ | string and/or integer |  |  |  |  |


---

Markdown generated with [jsonschema-markdown](https://github.com/elisiariocouto/jsonschema-markdown).
```

Also I receive a Warning, saying

```
2024-09-10 12:40:10.932 | WARNING  | jsonschema_markdown.converter.markdown:_handle_array_like_property:357 - Array-like property without type, selecting first type of subschema: string
```

Why is this?

fin swimmer

**Comment by elisiariocouto** [comment]
Hello @finswimmer, thank you for your feedback!
Indeed we have an unexpected behavior using your example. I remember that, with my initial tests, I didn't find a readable way to handle these situations. I guess that the output should have a "nullable" column or, in the type column, we can write `string` or `null` (using your example). What do you prefer? I'm leaning towards the second option, I think that too many columns are confusing to read.

**Comment by finswimmer** [comment]
Hey @elisiariocouto,

thanks a lot for your response. I'm not good at front-end stuff :sweat_smile: but I guess I'm also leaning towards the second option.

fin swimmer

**Comment by elisiariocouto** [comment]
Hello @finswimmer. Just released 0.3.10 which supports nullable fields. Give it a try and if you find any issue, feel free to comment. Thank you
